### ISSUE TITLE
Accessibility: Help text is provided as placeholder text

### ISSUE THREAD

**Issue by kemiga** [sky-uk/toolkit-ui#146]
## Expected Behavior
Help text should be provided as text adjacent to the corresponding form field and associated with the form field using "aria-describedby".

## Current Behavior
Help text is provided as placeholder text. Not all screen readers announce placeholder text, and the placeholder text has a very low contrast ratio (2.4:1) with the white background of the form field.

## Suggestion
Either the placeholder text should be removed and the help text should be provided in text adjacent to the form field, or the placeholder text should be given more contrast (at least 4.5:1 with the white background) and additional help text provided and then hidden offscreen whilst still accessible to screen readers. As stated before, help text adjacent to the form field should be associated with the form field using "aria-describedby".

**Comment by joebell93** [comment]
Thanks for this @kemiga I'm going to raise this as a design issue, and provide `aria-describedby` on https://github.com/sky-uk/toolkit-ui/issues/147 in our sample markup

**Comment by joebell93** [comment]
Contrast has increased thanks to the latest Normalize update, so I'll be closing this issue
